how-to: RESOLVE merge conflicts (#3929)
iterative/dvc.org: content/docs/user-guide/how-to/resolve-merge-conflicts.md

### Append-only directories

If you have an "append-only" dataset (where people only add new
files/directories) DVC provides a [Git merge driver] that can automatically
resolve merge conflicts into a parent branch. To use it, first set it up in your

> Git repo:

Author: @yathomasi did I express this correctly? Rel https://github.com/iterative/dvc.org/issues/3915#issuecomment-1234126026.

Reviewer: It looks like I was mistaken. https://github.com/iterative/dvc.org/issues/3915#issuecomment-1241890681

Author: NP, rolling back:
```suggestion
If you have an "append-only" dataset (where people only add new
files/directories) DVC provides a [Git merge driver] that can automatically
resolve merge conflicts for you. To use it, first set it up in your Git repo:
```
